### PR TITLE
Compiler: unfork prettier config

### PR DESCRIPTION
**Pull request by kassens**
Updates the prettier config to format all `.ts` and `.tsx` files in the repo using the existing defaults and removing overrides.

The first commit in this PR contains the config changes, the second is just the result of running `yarn prettier-all`.

### CODE REVIEW

**Review by poteto — CHANGES_REQUESTED** [review]
Hmm I get the desire to have consistent configs, but I'm wondering if this warrants a discussion? We've used the existing settings for a while now so there's some familiarity with how it was formatted before. Makes sense to be consistent, but styling is subjective so I think it'd be worth getting everyone's thoughts before landing this one.

Sending back to your queue though to run `yarn snap:build && yarn snap --update` to update the snapshots

**Review by poteto — APPROVED** [review]
I like our old config better but accepting to unblock, could you also add the commit hash to the skiplist in https://github.com/facebook/react/blob/main/.git-blame-ignore-revs in  once it lands?

**Comment by kassens** [comment]
> I like our old config better but accepting to unblock, could you also add the commit hash to the skiplist in https://github.com/facebook/react/blob/main/.git-blame-ignore-revs in once it lands?

I think you can argue to make a change to the full react repo style if there's a change you'd like to introduce. Otherwise, I think default should be what React was using for most of the code?

**Pull request merged**
